project.md

```diff
@@ -5,47 +5,65 @@
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
+	This DSL is aimed for those who want to create sheet music and chord progressions or melodies quickly, and who don't have that much experience with programming.
+
 ### What's the need?
 
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help
 them?_
 
+	I'm aiming to create a program that would let them quickly prototype things, without having to go through all the hurdles of designing things in a full music production application.
```

Comment:
I think this is a great idea because like you said, sometimes you just want to write something out on a scratch piece of paper instead of having to navigate a program with ten million bells and whistles. I think your challenge will be to avoid feature creeping and maintain a neat, concise language.